test/AxelarGateway.js

```diff
@@ -1156,105 +1450,163 @@ describe('AxelarGateway', () => {
         const decimals = 18;
         const cap = 1e9;
 
-        before(async () => {
-            await deployGateway();
+        describe('invalid token deployer', () => {
+            before(async () => {
+                await deployGateway(true);
+            });
+
+            it('should fail if token deployment fails', async () => {
+                const tokenSymbol = `TEST${getRandomString(10)}`;
+
+                const deployAndMintData = buildCommandBatch(
+                    await getChainId(),
+                    [getRandomID()],
+                    ['deployToken'],
+                    [getDeployCommand(tokenName, tokenSymbol, decimals, cap, ethers.constants.AddressZero, 0)],
+                );
+
+                const input = await getSignedWeightedExecuteInput(
+                    deployAndMintData,
+                    operators,
+                    getWeights(operators),
+                    threshold,
+                    operators.slice(0, threshold),
+                );
+
+                await gateway.execute(input, getGasOptions()).then((tx) => tx.wait());
+
+                const tokenAddress = await gateway.tokenAddresses(tokenSymbol);
+                expect(tokenAddress).to.equal(AddressZero);
+            });
         });
 
-        it('should burn internal token and emit an event', async () => {
-            const tokenSymbol = `TEST${getRandomString(10)}`;
+        describe('valid token deployer', () => {
```

Comment:
```suggestion
        describe('send token', () => {
```
this is testing send token it seems

Reply:
you can call it "negative send token test" maybe? Or since it's only one, we don't really need a separate describe block. Just feel like invalid token deployer isn't accurate in terms of what's being tested.